lib/mix/tasks/tower_rollbar.install.ex

```diff
@@ -0,0 +1,203 @@
+if Code.ensure_loaded?(Igniter) && Code.ensure_loaded?(Tower.Igniter) do
+  defmodule Mix.Tasks.TowerRollbar.Install do
+    @example "mix igniter.install tower_rollbar"
+
+    @shortdoc "Installs TowerRollbar. Invoke with `mix igniter.install tower_rollbar`"
+    @moduledoc """
+    #{@shortdoc}
+
+    ## Example
+
+    ```bash
+    #{@example}
+    ```
+    """
+
+    use Igniter.Mix.Task
+
+    import Tower.Igniter
+
+    @default_runtime_config """
+    import Config
+
+    if config_env() == :prod do
+      config :tower_rollbar,
+        access_token: System.get_env("ROLLBAR_SERVER_ACCESS_TOKEN"),
+        environment: System.get_env("DEPLOYMENT_ENV", to_string(config_env()))
+    end
+    """
+
+    @prod_config_patterns [
+      """
+      if config_env() == :prod do
+        __cursor__()
+      end
+      """,
+      """
+      if :prod == config_env() do
+        __cursor__()
+      end
+      """
+    ]
+
+    @rollbar_config_code """
+    config :tower_rollbar,
+      access_token: System.get_env("ROLLBAR_SERVER_ACCESS_TOKEN"),
+      environment: System.get_env("DEPLOYMENT_ENV", to_string(config_env()))
+    """
+
+    @impl Igniter.Mix.Task
+    def info(_argv, _composing_task) do
+      %Igniter.Mix.Task.Info{
+        group: :tower,
+        adds_deps: [],
+        installs: [],
+        example: @example,
+        only: nil,
+        positional: [],
+        composes: [],
+        schema: [],
+        defaults: [],
+        aliases: [],
+        required: []
+      }
+    end
+
+    @impl Igniter.Mix.Task
+    def igniter(igniter) do
+      igniter
+      |> add_reporter_to_config(TowerRollbar)
+      |> configure_runtime()
+    end
+
+    defp configure_runtime(igniter) do
+      if runtime_config_exists?(igniter) do
+        igniter
+      else
+        add_runtime_config(igniter)
+      end
+    end
```

Comment:
This function and it's several cascade called private functions seems they are going to be repeated on every reporter.

Candidates for generalization in `Tower.Ingiter` module.